plugins/MantisCoreFormatting/files/markdown.css

```diff
@@ -0,0 +1,10 @@
+/* quotes */
+.bug-description blockquote,
+.bug-steps-to-reproduce blockquote,
+.bug-additional-information blockquote,
+.bugnote-note blockquote {
```

Comment:
Targeted selectors are generally good, but in this case considering that Markdown text may be used in many places in Mantis, I'm quite certain there will be cases where this is too specific and will introduce display regressions because bootstrap CSS defines blockquote with font-size: 17.5px, see https://github.com/mantisbt/mantisbt/blob/6540bb1abecd9830b5705151ff25c3982ddec107/css/bootstrap-3.4.1.css#L1476-L1481

I didn't check for every possible use case, but the following are for sure displayed incorrectly:
- Project description (manage_proj_page.php)
- Version description (changelog_page.php / roadmap_page.php)
- textarea Custom fields (view.php page)
- Description, steps to reproduce, additional info, textarea CF on view_all_bug_page.php

Adding specific selectors for every single occurrence will be tedious and even not possible in some cases without changing the markup in some cases (e.g. manage_proj_page.php), so I'm wondering if we should not simply use `blockquote` without a class qualifier.